test/controllers/account/deletions_controller_test.rb

```diff
@@ -0,0 +1,65 @@
+require "test_helper"
+
+class DeletionsControllerTest < ActionDispatch::IntegrationTest
```

Comment:
In general, if we are testing the user interface, we should do that with system tests. These are more readable and maintainable than doing xpath-style selects on the raw documents.

The controller tests should be kept for testing edge case behaviour, e.g. when sending incorrect information to the controller, that the UI wouldn't normally allow. So the accounts_controller_test part is good, since it's checking that people can't post deletions directly to the controller.

I'm well aware that our current test suite has an awful lot of UI testing in controller tests, but for new features we should use system tests. We also have system tests already for account deletion (and not controller tests) so let's stick with that.

Reply:
moved tests from `test/controllers/account/deletions_controller_test.rb` to `test/system/account_deletion_test.rb`